src/main/java/org/team199/robot2021/Constants.java

```diff
@@ -235,6 +235,8 @@
             // intake/feeder
             public static final int kIntakeButton = X;
             public static final int kOuttakeButton = Y;
+            // limelight toggle
+            public static final int kLimelightToggleButton = 0; //TODO: set button
```

Comment:
Work with the driver to pick an unused button. I'm concerned that 0 might interfere with an existing button or cause a runtime error.